pages/blog/april-2019-update.md

```diff
@@ -0,0 +1,66 @@
+---
+description: Here are the most significant changes in April.
+---
+
+# April 2019 Update
+
+**Olivier Tassinari**
+
+*May 7, 2019*
+
+Here are the most significant changes in April:
+
+- 📚 We have migrated [most](https://github.com/mui-org/material-ui/issues/14897) of our demos to TypeScript. **@eps1lon** has lead the effort. You can switch between the JavaScript demo and the TypeScript demo using this toggle button:
+
+  ![TypeScript switch](/static/blog/april-2019-update/typescript.png)
+
+  This was only possible thanks to the contributions **@merceyz**, **@sperry94**, **@jasondashwang**, **@bh1505**, **@donigianrp**, **@Dudrie**, **@eluchsinger**, **@cahilfoley**, **@gabrielgene**, **@kenzhemir**, **@Adherentman**, **@lksilva**, **@Tevinthuku**. Thank you!
+  Supporting these TypeScript demos has one important implication, it forces us to have working TypeScript definitions ✨.
+- 🎀 We have migrated a large chunk of our components from classes to hooks. **@joshwooding** has lead the effort. We will explain why in the v4 release blog post.
+- 📐 We have updated our components to better match the Material Design specification
+  (Snackbar, List, Checkbox, Radio & Switch).
+- 🎁 We have added a demo for building a [TransferList](https://next.material-ui.com/demos/transfer-list/) component.
+
+  [![Transfer List](/static/blog/april-2019-update/transfer-list.png)](https://next.material-ui.com/demos/transfer-list/)
+
+- 💅 We have changed the class name generation to output global class names.
+  We have seen many people struggling with our `classes` API.
+  This API targets plain CSS and styled-components users.
+  It can be challenging to apply your class name on the right element with the `classes` API. It can also be cumbersome. ⚠️ Using global class names provide more power bug comes with a responsibility. We encourage any pattern that increases your **custom style isolation**.
```

Comment:
```suggestion
  It can be challenging to apply your class name on the right element with the `classes` API. It can also be cumbersome. ⚠️ Using global class names provide more power but comes with a responsibility. We encourage any pattern that increases your **custom style isolation**.
```